src/point_one/fusion_engine/messages/defs.h

```diff
@@ -295,6 +295,7 @@ P1_CONSTEXPR_FUNC bool IsCommand(MessageType message_type) {
   switch (message_type) {
     case MessageType::MESSAGE_REQUEST:
     case MessageType::RESET_REQUEST:
+    case MessageType::STARTUP_REQUEST:
```

Comment:
It's a little bit annoying, but to avoid this problem in the future, you should probably restructure this switch to not have a `default`. If you explicitly handle each case in the switch, the compiler will give a warning if we add a new type and don't update this function. For extra safety you can still return false outside the switch.